### PR TITLE
feat(front): hide subtile with unknown status in group

### PR DESCRIPTION
**Pull request by jsdidierlaurent**

### CODE REVIEW

File: front/src/components/Tile.vue [hunk 1]
```diff
@@ -155,20 +163,20 @@ import { TileValueUnit } from '@/store'
       return this.$store.getters.theme.toString().toLowerCase()
     }
 
-    get nonSucceededSubTiles(): TileConfig[] | undefined {
+    get displayableSubTiles(): TileConfig[] | undefined {
```
Comment (Alex-D) [inline front/src/components/Tile.vue]: `displayedSubTiles`

File: front/src/components/Tile.vue [hunk 2]
```diff
@@ -155,20 +163,20 @@ import { TileValueUnit } from '@/store'
...
       }
 
-      const nonSucceededSubTiles = this.config.tiles.filter((subTile) => {
+      const displayableSubTiles = this.config.tiles.filter((subTile) => {
         if (!this.$store.state.tilesState.hasOwnProperty(subTile.stateKey)) {
           return false
         }
 
-        return this.$store.state.tilesState[subTile.stateKey].status !== TileStatus.Success
+        return DISPLAYABLE_SUBTILE_STATUS.includes(this.$store.state.tilesState[subTile.stateKey].status)
```
Comment (Alex-D) [inline front/src/components/Tile.vue]: Keep DISPLAYABLE

File: front/src/components/Tile.vue [hunk 3]
```diff
@@ -53,7 +53,15 @@ import { TileValueUnit } from '@/store'
...
   import MonitororTileIcon from '@/components/TileIcon.vue'
-  import {TileAuthor, TileConfig, TileState, TileStatus, TileType, TileValueUnit} from '@/store'
+  import {
+      DISPLAYABLE_SUBTILE_STATUS,
+      TileAuthor,
+      TileConfig,
+      TileState,
+      TileStatus,
+      TileType,
+      TileValueUnit,
```
Comment (Alex-D) [inline front/src/components/Tile.vue]: Why all those imports here?

File: front/src/store.ts [hunk 4]
```diff
@@ -267,13 +275,13 @@ const store: StoreOptions<RootState> = {
           return mostImportantStatus(worstSubTileStatus, subTileStatus)
         }, TileStatus.Unknown)
 
-        const groupSucceededSubTiles = groupSubTilesState.filter((subTileState) => {
+        const groupNonDisplayableSubTiles = groupSubTilesState.filter((subTileState) => {
```
Comment (Alex-D) [inline front/src/store.ts]: `groupNonDisplayedSubTiles`

File: front/src/components/Tile.vue (hunk 3 above)
Comment (jsdidierlaurent) [reply front/src/components/Tile.vue]: They were already there, they are just posted in multiline now

**Pull request merged**